proof/taproot.go

```diff
@@ -579,3 +580,64 @@ func (p TaprootProof) DeriveByTapscriptProof() (*btcec.PublicKey, error) {
 	}
 	return p.TapscriptProof.DeriveTaprootKeys(p.InternalKey)
 }
+
+// AddExclusionProofs adds exclusion proofs to the base proof for each P2TR
+// output in the given PSBT that isn't an anchor output itself. To determine
```

Comment:
Took me some time to realize this was not "anchor outputs" as in the LN context 😅 Would perhaps be beneficial to name them something distinct

Reply:
Heh yeh, we've been using "anchor" here to mean the output that all the Taro commitments are rooted in.